lib/whitehall_importer/integrity_checker.rb

```diff
@@ -27,7 +31,9 @@ def content_problems
          document_type
          schema_name).each do |attribute|
         if publishing_api_content[attribute] != proposed_payload[attribute]
-          problems << "#{attribute} doesn't match"
+          problems << "#{attribute} doesn't match, " \
+            "expected: #{publishing_api_content[attribute].inspect}, " \
+            "actual: #{proposed_payload[attribute].inspect}"
```

Comment:
Might want to put this into a method or class where we pass in message, expected and actual and that can convert this to a string.

Reply:
Yeah. I had the same thought, but hadn't gotten around to it yet.